### PR TITLE
Appveyor wheel change

### LINKED ISSUE #2488: Appveyor build configured to download nupic.bindings from S3
The C++ extension exists in `nupic.bindings`, a pip package that should exist on S3. This can be installed with `pip install <URI to binary file>`.

Depends on https://github.com/numenta/nupic.core/issues/570.

### LINKED ISSUE #2814: Nupic Windows wheel is named incorrectly
Plus requires a change to cope with differing nupic.bindings wheel name after the big CMake merge in nupic.core

### PR DESCRIPTION
**Pull request by rcrowder**
Fixes #2814 
Fixes #2488
Depends on https://github.com/numenta/nupic.core/pull/741

### CODE REVIEW

**Comment by rcrowder** [comment]
@scottpurdy This is ready, but requires the new wheel from nupic.core on S3. I'll leave this marked as In Progress, and update status and ping you when S3 has the right bindings wheel.

**Comment by rcrowder** [comment]
https://github.com/numenta/nupic/pull/2813 Also needs resolving before tests will all pass, and the wheel building stage kicks in.

**Comment by rcrowder** [comment]
The closed #2813 work has now been placed in the dependent numenta/nupic.core#741 PR

**Comment by rhyolight** [comment]
:+1: when green

**Comment by rcrowder** [comment]
@rhyolight cc @scottpurdy We have a green light on Travis and AppVeyor. Ready for review.

**Comment by rcrowder** [comment]
@rhyolight The S3 directory looks wrong to me. Because it include the commit sha. I think this should just be `folder: "numenta/nupic` ?

**Comment by rcrowder** [comment]
Update: It seems like the right location, considering there is no PyPi support yet. The sha inclusion allows for a differentiation of varying nupic wheel versions on S3.

File: appveyor.yml [hunk 1]
```diff
@@ -84,8 +93,14 @@ after_build:
   # Show nupic installation folder by trying to import nupic, if works, it prints
   # the absolute path of nupic.__file__, which the installation folder itself.
   - python -c "import sys;import os;import nupic.data;sys.stdout.write(os.path.abspath(os.path.join(nupic.data.__file__, '../..')) + '\n')"
+
+  # Setup local USER name
   - set USER=appveyor
 
+  # Copy a custom AppVeyor site XML for OPF database connections,
+  # into local USER site-packages directory
+  - copy ci\appveyor\nupic-site.xml C:\Users\appveyor\AppData\Roaming\Python\Python27\site-packages\nupic\support\nupic-site.xml
```
Comment (scottpurdy) [inline appveyor.yml]: Can we just use `src/nupic/support/nupic-default.xml`? Why create a copy of that at `ci/appveyor/nupic-site.xml`?

**Comment by scottpurdy** [comment]
:+1: aside from adding a copy of the xml config file.

File: appveyor.yml (hunk 1 above)
Comment (rcrowder) [inline appveyor.yml]: I'll have a look and see how to change the password entry in the `nupic-default.xml` so the `ci/appveyor/nupic-site.xml` can be deleted. Or work out whether it's possible to change the MySQL password on AppVeyor and restart the service. Although the former option maybe the easiest.

Comment (rcrowder) [inline appveyor.yml]: I've added an alternative way of changing the XML. Waiting on AV now to see if it works; https://ci.appveyor.com/project/numenta-ci/nupic/build/0.1.0.335

**Comment by rcrowder** [comment]
@scottpurdy cc @rhyolight My last change to use powerscript to edit the XML of a copied nupic-default.xml has worked on AppVeyor.

Ready for final review/merge.

**Pull request merged**